prql-compiler/snap/snapcraft.yaml

```diff
@@ -0,0 +1,25 @@
+name: prqlc # you probably want to 'snapcraft register <name>'
```

Comment:
If possible, can we put this file into `/.config/snap`? 

If that's not possible somewhere in `prql-compiler/prqlc`, since we're publishing `prqlc`.

Reply:
It seems that wouldn't be possible.
https://github.com/so-fancy/diff-so-fancy/pull/434

Reply:
Thanks for looking into it. Possibly we should move into `prql-compiler/prqlc`, but we can change that later if that's easier